.github/workflows/monthly-build-deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: monthly-build-deploy
+
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 1 * *'
+
+env:
+  AWS_DEFAULT_REGION: us-east-1
+  AWS_DEFAULT_OUTPUT: json
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        accountnumber: [{{ secrets.AWS_ACCOUNT_VOTESHIELD }}, {{ secrets.AWS_ACCOUNT_DEV }}]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Login to ECR
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ matrix.accountnumber }}.dkr.ecr.us-east-1.amazonaws.com
+
+      - name: Build and push
+        id: docker_build_base
+        uses: docker/build-push-action@v2
+        with:
+          context: pritunl-zero-ecs-bastion
+          push: true
+          tags: ${{ matrix.accountnumber }}.dkr.ecr.us-east-1.amazonaws.com/pritunl-zero-ecs-bastion:latest
```

Comment:
If we are building and deploying here without testing, it might be good to add a Slack notification here just to help with making sure the update is visible in case something does happen.

Reply:
That's an interesting idea, and I like it. I will file an issue and we can discuss it more there, I think